src/composables/resetAccountWithFirebase.js

```diff
@@ -0,0 +1,27 @@
+import { getAuth, updateEmail, updatePassword } from "firebase/auth"
```

Comment:
there's already an named export that is firebase configured, use that not `getAuth`